src/njoy21/input/POWR/Lib3/src/readCard10.hpp

```diff
@@ -0,0 +1,16 @@
+template< typename Istream >
+std::vector< optional< Card10 > > readCard10( Istream& is,
+                       const std::vector< Card5 > card5List, const int iclaps ){
+  std::vector< optional< Card10 > > card10List;
+
+  int nres = ( iclaps == 1 ) ? 15 : 13;
+
+  for( auto card5 : card5List ){
+    if(card5.nina.value == 0 or card5.nina.value == 3 ){
```

Comment:
What about if `ires=1`?

Reply:
Based on the input instructions, it's impossible for ires to even be read without nina = 0 or 3, so it is redundant.